src/QMCDrivers/DMC/DMCBatched.cpp

```diff
@@ -51,12 +62,19 @@ QMCTraits::IndexType DMCBatched::calc_default_local_walkers(IndexType walkers_pe
   return local_walkers;
 }
 
+void DMCBatched::setNonLocalMoveHandler(QMCHamiltonian& golden_hamiltonian)
+{
+  golden_hamiltonian.setNonLocalMoves(dmcdriver_input_.get_non_local_move(), qmcdriver_input_.get_tau(),
```

Comment:
Not understanding why only golden_hamiltonian gets set. This breaks the requirement that walker in a crowd must have identical state.